accel/tcg/log_instr.c

```diff
@@ -416,12 +426,16 @@ static void emit_text_stop(CPUArchState *env, target_ulong pc)
 {
     cpu_log_instr_state_t *cpulog = get_cpu_log_state(env);
 
-    if (qemu_loglevel_mask(CPU_LOG_USER_ONLY) && cpulog->user_mode_tracing)
-        qemu_log("User-mode only tracing disabled at " TARGET_FMT_lx
-                 ", ASID %u\n", pc, cpu_get_asid(env));
-    else
-        qemu_log("Disabled instruction logging @ " TARGET_FMT_lx
-                 " ASID %u\n", pc, cpu_get_asid(env));
+    switch (cpulog->loglevel) {
```

Comment:
Here and above: I'm not sure using a switch for only one case is ideal. Unless you are planning to add more in the future?